apps/web/components/dashboard/bookmarks/TagsEditor.tsx

```diff
@@ -120,14 +120,32 @@ export function TagsEditor({ bookmark }: { bookmark: ZBookmark }) {
           overflowY: "auto",
           scrollbarWidth: "none",
         }),
+        container: (styles) => ({
+          ...styles,
+          width: "100%",
+        }),
         control: (styles) => ({
           ...styles,
+          overflow: "hidden",
           backgroundColor: "hsl(var(--background))",
           borderColor: "hsl(var(--border))",
           ":hover": {
             borderColor: "hsl(var(--border))",
           },
         }),
+        input: (styles) => ({
+          ...styles,
+          color: "rgb(156 163 175)",
+        }),
+        menu: (styles) => ({
+          ...styles,
+          overflow: "hidden",
+          color: "rgb(156 163 175)",
+        }),
+        placeholder: (styles) => ({
+          ...styles,
+          color: "hsl(var(--muted-foreground))",
+        }),
```

Comment:
One day, I'll get rid of this ugly `react-select` component and all the hacks I did to make look like the rest of our components :D